packages/core-database-postgres/src/migrations/index.ts

```diff
@@ -13,4 +13,5 @@ export const migrations = [
     loadQueryFile(__dirname, "./20181204400000-add-recipient_id-index-to-transactions-table.sql"),
     loadQueryFile(__dirname, "./20190307000000-drop-wallets-table.sql"),
     loadQueryFile(__dirname, "./20190313000000-add-asset-column-to-transactions-table.sql"),
+    loadQueryFile(__dirname, "./20190606000000-add-block-id-fk-on-transactions.sql"),
```

Comment:
No need to abbreviate the name - `add-block-id-foreign-key-on-transactions`

Reply:
Done in 5f8f7a563.